.gitignore

```diff
@@ -16,3 +16,6 @@ bower_components
 # Xcode
 
 xcuserdata
+
+# mac
+.DS_Store
```

Comment:
I think that should be removed because it is only related to your local environment

Reply:
you are right, corrected.